docs/Solidity.g4

```diff
@@ -53,8 +53,30 @@ contractPart
   | eventDefinition
   | enumDefinition ;
 
+//natSpec : WS* ;
```

Comment:
Do we really want to make this a part of the grammar? It might confuse some tools

Reply:
you mean Natspec or more precise distintuishment of whitespace vs newlines?

I gave up on natspec inside the .g4 file for now, because I ended up reading too much of Antlr documentation without direct success. I wanted to revisit the Antlr syntax at the end again.